src/main/scripts/summa_export/get_records.sh

```diff
@@ -58,15 +63,28 @@ get_record() {
     echo " - Fetching $OUT_FOLDER/$OUT_NAME"
     if [[ "true" == "$EXPAND" ]]; then
         # expand is ignored
-        curl -s -G "$SUMMA_STORAGE?method=getCustomRecord&expand=false&legacyMerge=false&escapeContent=false&" --data-urlencode "id=${RECORD_ID}" | sed -e 's/<soapenv.*getCustomRecordReturn[^>]*>//' -e 's/<\/ns1:getCustomRecordReturn.*//' -e 's/&lt;/</g' -e 's/&gt;/>/g' -e 's/&quot;/"/g' -e 's/&amp;/&/g' | xmllint --format - > "$OUT_FOLDER/$OUT_NAME"
+
+        # Fetch the raw response (entity escaped record XML wrapped in a SOAP envelope)
+        local RESPONSE=$(curl -s -G "$SUMMA_STORAGE?method=getCustomRecord&expand=false&legacyMerge=false&escapeContent=false&" --data-urlencode "id=${RECORD_ID}")
+
+        # Check that there is content inside of the SOAP envelope
+        if [[ "." != ".$( tr '\n' ' ' <<< "$RESPONSE" | grep "<ns1:getCustomRecordReturn[^>]*>.\{10,\}<\/ns1:getCustomRecordReturn")" ]]; then
+            echo "$RESPONSE" | sed -e 's/<soapenv.*<ns1:getCustomRecordReturn[^>]*>//' -e 's/<\/ns1:getCustomRecordReturn.*//' -e 's/&lt;/</g' -e 's/&gt;/>/g' -e 's/&quot;/"/g' -e 's/&amp;/&/g' | xmllint --format - > "$OUT_FOLDER/$OUT_NAME"
```

Comment:
Note: The sed-expression is still prone to mismatching in case of an autoclosed xml-tag.
However, the check above should ensure that this is not the case, so it's not a problem as such.